main.js

```diff
@@ -1,3 +1,44 @@
+// We need the second hand of the clock to rotate 6 degrees every second.
+   // 'rotate(6deg)
+   // function rotateHand()
+   // hand.style.transform = `rotate(6
+   // setInterval(rotateHand(), 1000); for 1000ms
+   // 
+
+
+
+// variable for the hand
 const hand = document.querySelector(".hand");
 
-hand.style.transform = `rotate(90deg)`; //🔍 HINT: THis is an example of how to rotate the hand!
+// variable for minute
+const minute = document.querySelector(".minute");
+
+ // function rotating the hand 6deg
+let degrees = 0;
```

Comment:
Rather than starting degrees at 0, we could start it at  `(new Date().getSeconds()) * 6`, so that it matches the current time. We could do the same for minutes too.